common/src/main/java/com/wynntils/services/mapdata/MapFeaturePoiWrapper.java

```diff
@@ -63,17 +64,22 @@ public String getName() {
 
     @Override
     public int getWidth(float mapZoom, float scale) {
-        String iconId = attributes.getIconId();
-        String label = attributes.getLabel();
+        Optional<String> iconId = attributes.getIconId();
```

Comment:
I have the feeling that these might be pushed down to much closer their actual use.